### PR TITLE
Un-broke the menu pages. Removed usage of deprecated ListView.

### PR DESCRIPTION
**Pull request by jameslu123**

### CODE REVIEW

**Comment by ygao76** [comment]
Add description and testing method

File: restaurant-app-template/RestaurantApp/App/Containers/Menu.js [hunk 1]
```diff
@@ -101,24 +100,27 @@ class Menu extends React.Component {
...
-          description:theReceivedItems[i].description,
-          photo:{ uri: theReceivedItems[i].photo }
-          //_key: child.key
-        });
+        var numFoodItemsForCategory = 0;
+        if (theReceivedItems[i].items !== undefined)
+        {
+          numFoodItemsForCategory = theReceivedItems[i].items.length;
+        }
+        for (var j=0; j<numFoodItemsForCategory; j++)
```
Comment (ygao76) [inline restaurant-app-template/RestaurantApp/App/Containers/Menu.js]: use forEach

Comment (jameslu123) [reply restaurant-app-template/RestaurantApp/App/Containers/Menu.js]: ok

Comment (jameslu123) [reply restaurant-app-template/RestaurantApp/App/Containers/Menu.js]: Hard to use foreach in javascript, because it is an array specific function.

**Pull request merged**